pkgs/development/python-modules/fastpbkdf2/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage
+, openssl, pytest, cffi, six }:
+
+buildPythonPackage rec {
+  pname = "fastpbkdf2";
+  version = "0.2";
+  src = fetchFromGitHub {
```

Comment:
Why not `fetchPypi`?

Reply:
The package has tests on Github, but not on PyPI. In cases such as this, is it still better to switch to PyPI?

Reply:
In that case, it's okay to use GitHub. However, please add a comment with the reason.

Reply:
Okay, I added a comment explaining the choice.